app/controlplane/internal/service/attestation.go

```diff
@@ -470,9 +470,19 @@ func extractPolicyEvaluations(in map[string][]*chainloop.PolicyEvaluation) map[s
 			}
 
 			if ev.PolicyReference != nil {
+				name, _ := ev.PolicyReference.GetAnnotations().AsMap()["organization"].(string)
```

Comment:
This won't work with old attestations right? Could we add a check to only add the organization name if the fields exists?

Reply:
If the key doesn't exist, or there are no annotations, `orgName` will be assigned an empty string `""`. I've tested it with references without groups and works as expected.